frontend/components/inbox/TicketCard.vue

```diff
@@ -1,59 +1,100 @@
 <template>
-  <card class="p-2" outlined v-bind:class="{ 'py-1': small }">
-    <span class="text-xs text-gray-500 float-right">
-      #{{ ticket.ticket_inbox_id }}
-    </span>
+  <div
+    class="relative rounded overflow-hidden bg-white p-2 border"
+    v-bind:class="{ 'py-1': small, 'border-primary': assignee.username == user.username }"
+  >
+    <div class="flex justify-between items-start pb-1">
+      <router-link
+        :to="link"
+        class="font-medium leading-4 text-gray-900 hover:underline"
+      >
+        {{ ticket.title }}
+      </router-link>
 
-    <h3 v-if="!small" class="text-xs text-gray-500">{{ date(ticket.date_created) }}</h3>
-    <h3 class="text-xs text-gray-500" v-if="!small && ticket.assignee && ticket.assignee.username">
-      Assignee: {{ticket.assignee.first_name }} {{ ticket.assignee.last_name }}
-    </h3>
+      <span class="text-xs text-gray-500 float-right">
+        #{{ ticket.ticket_inbox_id }}
+      </span>
+    </div>
 
-    <router-link :to="link" class="font-medium text-gray-900 hover:underline">
-      {{ ticket.title }}
-    </router-link>
+    <div class="flex justify-between items-center">
+      <h3 v-if="!small" class="text-xs text-gray-500">
+        <span class="font-medium">{{ ticket.author.first_name }} {{ ticket.author.last_name }}</span>・{{ date(ticket.date_created) }}
+      </h3>
+      <div v-if="ticket.labels.length == 0 && assignee.avatar_url" class="flex-shrink-0">
+        <img class="h-5 w-5 rounded-full" :src="assignee.avatar_url" alt="" :title="assignee.first_name + ' ' + assignee.last_name" />
+      </div>
+    </div>
+    <!-- TODO: Implement easy assign button. -->
+    <!-- <h3
+      v-if="!ticket || !assignee || !assignee.username"
+      class="text-xs text-gray-500"
+      >Assignee: No one—<button class="hover:text-orange-600 no-underline">
+        Assign yourself
+      </button></h3
+    > -->
+    <!-- <h3
+      class="text-xs text-gray-500"
+      v-if="!small && assignee && assignee.username"
+    >
+      {{ ticket.author.first_name }} {{ ticket.author.last_name }}
+    </h3> -->
```

Comment:
Dit kan weg denk ik

Reply:
I want to implement this later, but it needs some backend code, so the frontend is there already just in case.